src/components/DirectPaymentCreateModal/styles.scss

```diff
@@ -147,4 +151,75 @@
     color: var(--sds-clr-gray-09);
     font-weight: 500;
   }
+
+  &__confirmation {
+    padding: pxToRem(20px) 0;
+  }
+
+  &__confirmationWarning {
+    display: flex;
+    align-items: flex-start;
+    gap: pxToRem(8px);
+    color: var(--color-warning);
+    font-size: pxToRem(14px);
+    font-weight: 500;
+    margin-bottom: pxToRem(20px);
+    padding: pxToRem(12px);
+    background-color: var(--color-warning-10);
+    border: 1px solid var(--color-warning-30);
+    border-radius: pxToRem(4px);
+
+    p {
+      margin: 0;
+      flex: 1;
+    }
+  }
+
+  &__confirmationDetails {
+    display: flex;
+    flex-direction: column;
+    gap: pxToRem(12px);
+    padding: pxToRem(16px);
+    background-color: var(--color-gray-05);
+    border: 1px solid var(--color-gray-20);
+    border-radius: pxToRem(6px);
+  }
+
+  &__confirmationRow {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: pxToRem(8px) 0;
+    border-bottom: 1px solid var(--color-gray-20);
+
+    &:last-child {
+      border-bottom: none;
+    }
+  }
+
+  &__confirmationLabel {
+    font-weight: 500;
+    color: var(--color-gray-70);
+    font-size: pxToRem(14px);
+  }
+
+  &__confirmationValue {
+    font-weight: 600;
+    color: var(--color-gray-80);
```

Comment:
CSS variables starting with `--color-` are no longer valid in new SDS. All color variables now start with `--sds-clr-`. Best way to view what's available is by selecting `#root` div in dev tools. We might not have a 1:1 match from old to new, so pick the closest color (or check what is used in similar cases).

<img width="1017" height="1211" alt="image" src="https://github.com/user-attachments/assets/5d873141-62c8-4161-be84-cbc58d462686" />